src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Properties/InterceptedProjectProperties/ApplicationPropertyPage/PropertyNames.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+namespace Microsoft.VisualStudio.ProjectSystem.Properties;
+
+    /// <summary>
+    /// Contains constants representing the property names used in the ApplicationPropertyPage folder.
+    /// </summary>
+    internal class PropertyNames
```

Comment:
A few points about this type:

- It can be a `static class`
- The `Property` suffix can be removed from all field names as that is clear from their containing type
- Some of the property names don't match their values, such as `UseApplicationFramework`, `VBAuthenticationMode`, and all of the MSBuild properties (how do they differ from other project properties?). To fix this, consider using `nameof(...)` to ensure they're the same on both sides and prevent confusion.